drivers/modem/hl7800.c

```diff
@@ -1074,6 +1074,22 @@ int32_t mdm_hl7800_get_operator_index(void)
 	}
 }
 
+/**
+ * @brief Perform a site survey.
+ *
+ */
+int32_t mdm_hl7800_perform_site_survey(void)
+{
+	int ret;
+
+	hl7800_lock();
+	wakeup_hl7800();
+	ret = send_at_cmd(NULL, "at%meas=\"97\"", MDM_CMD_SEND_TIMEOUT, 0, false);
+	allow_sleep(true);
+	hl7800_unlock();
+	return ret;
```

Comment:
Indentation looks wrong here

Reply:
Has something changed in github or is it my browser. I am not seeing tab as indentation in the review.

Reply:
I think it must be something with your browser. I check all the files, spots you flagged. All are using tabs.
![image](https://user-images.githubusercontent.com/4430317/131678812-2924273c-408c-4ae6-999f-e9e3f26ceb36.png)